usecases/SimpleOntology/Mapping/SimpleOntologyMappedProbe1.ttl

```diff
@@ -32,9 +32,13 @@ xsd:float a rdfs:Datatype .
 
 xsd:string a rdfs:Datatype .
```

Comment:
not sure if the changes in these files here are on purpose, I guess they are not.

Reply:
I will have a look, might have been me, when I merged from main together with Atul

Reply:
Fixed it